UI/src/view/MassTransitStopForm.js

```diff
@@ -263,6 +270,7 @@
           elementType = property.propertyType === 'long_text' ?
             $('<textarea />').addClass('form-control') : $('<input type="text"/>').addClass('form-control');
           element = elementType.bind('input', function(target){
+            ValidatePlatformNumberMaxSize(target, property.publicId);
```

Comment:
We are binding this for all the inputs!? Since we have the if for the properties probably it could be bind only when we are creating the platform input.